### ISSUE TITLE
Search Message 

### ISSUE THREAD

**Issue by UncleHanzi** [Dixin/Etymology#100]
Case: Search Message 

Feedback from my friends is definitely:  
we want to make things as simple as possible for users

!This message is good for diagnostics, 

 𪛖: Found 0 etymologies and 0 characters in 0.298 seconds. Unicode standard: U+2A6D6  .

but 
Not necessary if we find the character and show its information elsewhere.

Case 1: If the Character is found show no message, do not be redundant 

Case 2: If Character is not found, 
show Unicode information and a message that says 
 “Rare character not yet analyzed” or something like that. 
Show this information in the same place you would show the other information. Not as a special message.

Also, there may be some cases where you find and display oracle, bronze, seal, showen information for which I have not analyzed information in my database table. 

In other words, It is possible I may have some rare character in my seal, oracle, bronze, lst tables that I have not explained yet in my etymology table
